### ISSUE TITLE
Is there any tests or comparison of dotvvm with other front-end frameworks?

### ISSUE THREAD

**Issue by konstantin-mvr** [riganti/dotvvm#599]
Hi, I'm writing a thesis, where I use dotvvm + asp.net core. I would like to write a chapter where I compare dotvvm for example with angular. Are somewhere any tests ?

Thank you.

**Comment by Mylan719** [comment]
If I remember correctly @exyi made series of performance tests comparing MVVM a MVC applications when dealing with rendering large amounts of data. And potentialy some other tests.

**Comment by exyi** [comment]
No, I haven't. I'm not aware of any application written both in DotVVM and Angular for comparison.

**Comment by konstantin-mvr** [comment]
@Mylan719  @exyi  so, do you know guys how to make some performance tests of framework?

**Comment by exyi** [comment]
On Wed, 2018-05-16 at 19:59 +0000, konstantin-mvr wrote:
> @Mylan719  @exyi  so, do you know guys how to make some performance tests of framework?

Hmm, kind of. This is a tricky problem - we have a bunch of tests at https://github.com/riganti/dotvvm-benchmarks/ that are used for comparison between DotVVM version.
However, we don't have any reasonable comparison between Asp.Net MVC or nay other framework, since the performance hugely depends on the way the application is written -
how much was the programmer willing to optimize things. Testing highly optimized usage of MVC/DotVVM does not make much sense as noone actualy does that, and testing
idiomatic usage of the framework also does not make much sense, as it's undecidable what's the "idiomatic" usage...

So, yes, we have *some* performance test, but we don't have any comparison (and have a reason for that).

**Comment by konstantin-mvr** [comment]
@exyi thanks:)
